Cargo.toml

```diff
@@ -157,7 +157,7 @@ socket2 = "0.5"
 strum = "0.24"
 strum_macros = "0.24"
 "subtle"  = "2.5.0"
-svg_fmt = "0.4.1"
+svg_fmt = { git = "https://github.com/neondatabase/fork--nical--rust_debug", branch = "neon" }
```

Comment:
Have you filed PRs upstream for the suggested features? The maintainer is very responsive.

Reply:
To add to this: the fork is already out of date compared to upstream's master. Following unchanged dependencies is always better than forking and carrying a patch, simply from a maintainability point of view. Latter should be temporary, if possible.

Reply:
> Have you filed PRs upstream for the suggested features? The maintainer is very responsive.

Yeah, it's linked in the PR description.

Reply:
I added a link to Cargo.toml as well, so that future readers of Cargo.toml are pointed to it: https://github.com/neondatabase/neon/pull/7455/commits/98a7820048c69c0c1d585ba0c5e3495c4ef71de9